pkg/component/monitoring/charts/seed-monitoring/charts/alertmanager/templates/alertmanager.yaml

```diff
@@ -1,175 +0,0 @@
-apiVersion: v1
-kind: Service
-metadata:
-  name: alertmanager-client
-  namespace: {{ .Release.Namespace }}
-  labels:
-    component: alertmanager
-    role: monitoring
-  annotations:
-    networking.resources.gardener.cloud/from-all-scrape-targets-allowed-ports: '[{"port":9093,"protocol":"TCP"}]'
-spec:
-  ports:
-  - port: 9093
-    name: metrics
-  type: ClusterIP
-  selector:
-    component: alertmanager
-    role: monitoring
----
-apiVersion: v1
-kind: Service
-metadata:
-  name: alertmanager
-  namespace: {{ .Release.Namespace }}
-  labels:
-    component: alertmanager
-    role: monitoring
-spec:
-  ports:
-  - port: 9093
-    name: cluster
-  type: ClusterIP
-  # This is important! Without it the mech won't work.
-  clusterIP: None
```

Comment:
This seems to be important in the past, but I don't find anything similar in the new version.
Does the operator create the service by default the right way?

Reply:
Yep, it does :) ref https://github.com/gardener/gardener/pull/9159#discussion_r1492242630

Reply:
There is already an answer for everything, thanks 😄